sa/model.go

```diff
@@ -403,14 +403,14 @@ func modelToOrder(om *orderModel) (*corepb.Order, error) {
 
 var challTypeToUint = map[string]uint{
 	"http-01":     0,
-	"tls-sni-01":  1,
+	"tls-sni-01":  1, // deprecated
```

Comment:
Given this code isn't currently in use I think this can just be removed outright.